feat: add issue template

minhyeong112 opened the pull request:
Resolves #128

minhyeong112 committed:
feat: add issue template

Resolves #128

0x4007 commented on .github/ISSUE_TEMPLATE/default.md:
```diff
@@ -0,0 +1,23 @@
+
+[description of link](url of any relevant comment, document, website, etc.)
+[description of link](url of any relevant comment, document, website, etc.)
+[description of link](url of any relevant comment, document, website, etc.)
```
```suggestion
- [description of link](https://my-example-link.com)
- [description of link](https://my-example-link.com)
- [description of link](https://my-example-link.com)
```

0x4007 reviewed (commented):
Did you test if this works as is? You can post on another repo and I can attempt to create a new issue for QA

minhyeong112 commented:
> Did you test if this works as is? You can post on another repo and I can attempt to create a new issue for QA

yes, feel free to try here: [github.com/minhyeong112/mig/issues](https://github.com/minhyeong112/mig/issues)

0x4007 commented:
> > Did you test if this works as is? You can post on another repo and I can attempt to create a new issue for QA
> 
> yes, feel free to try here: [github.com/minhyeong112/mig/issues](https://github.com/minhyeong112/mig/issues)

Oh I thought you were making a form which is better for the use case I think. Like here https://github.com/ubiquity/ubiquibar/issues/new?template=ubiquitee-order-wizard.yml

minhyeong112 committed:
refactor: convert issue template to form format

Converts the issue template to a more intuitive YAML form format.
Resolves #128

minhyeong112 commented:
> form

nice! will have one ready shortly

minhyeong112 committed:
refactor: rename and update issue template

Renamed to 'BizDev Task Proposal Template' and incorporated feedback changes.

minhyeong112 committed:
update: incorporate feedback tweaks to issue template

minhyeong112 commented:
Done
Please [check here](https://github.com/minhyeong112/mig/issues/new?template=BizDev+Task+Proposal+Template.yml) and let me know if you want any changes~

0x4007 reviewed (approved):
<img width="470" alt="image" src="https://github.com/user-attachments/assets/b0a8fe57-4b02-4221-9c04-7579adad0759" />

This is funny but I couldn't test presumably because I am not a collaborator on the `mig` repo
